rmgpy/data/thermo.py

```diff
@@ -1295,53 +1295,57 @@ def get_thermo_data(self, species, metal_to_scale_to=None, training_set=None):
+                        # Too many radicals for direct calculation: use HBI.
+                        logging.info("{0} radicals on {1} exceeds limit of {2}. Using HBI method.".format(
+                            original_molecule.get_radical_count(),
+                            species.label,
+                            quantum_mechanics.settings.maxRadicalNumber,
+                        ))
+
+                        # Need to estimate thermo via each resonance isomer
+                        thermo = []
+                        for molecule in species.molecule:
+                            molecule.clear_labeled_atoms()
+                            # Try to see if the saturated molecule can be found in the libraries
+                            tdata = self.estimate_radical_thermo_via_hbi(molecule, self.get_thermo_data_from_libraries)
+                            priority = 1
+                            if tdata is None:
+                                # Then attempt quantum mechanics job on the saturated molecule
+                                tdata = self.estimate_radical_thermo_via_hbi(molecule, quantum_mechanics.get_thermo_data)
+                                priority = 2
+                            if tdata is None:
+                                # Fall back to group additivity
+                                tdata = self.estimate_thermo_via_group_additivity(molecule)
+                                priority = 3
+
+                            thermo.append((priority, tdata.get_enthalpy(298.), molecule, tdata))
+
+                        if len(thermo) > 1:
+                            # Sort thermo first by the priority, then by the most stable H298 value
+                            thermo = sorted(thermo, key=lambda x: (x[0], x[1]))
+                            for i, therm in enumerate(thermo):
+                                logging.debug("Resonance isomer {0} {1} gives H298={2:.0f} J/mol"
+                                              "".format(i + 1, therm[2].to_smiles(), therm[1]))
+                            # Save resonance isomers reordered by their thermo
+                            species.molecule = [item[2] for item in thermo]
+                            original_molecule = species.molecule[0]
+                        thermo0 = thermo[0][3]
+
+                        # update entropy by symmetry correction
+                        thermo0.S298.value_si -= constants.R * math.log(species.get_symmetry_number())
+
+                    else:  # Not too many radicals: do a direct calculation.
+                        thermo0 = quantum_mechanics.get_thermo_data(original_molecule)  # returns None if it fails
+            except ValueError: #rdkit fails to generate conformers 
+                logging.error("Quantum Mechanics calculation failed for species: {0}".format(species.label))
+                logging.error("Falling back to ML (If turned on) or GAV (If not)")
```

Comment:
ValueError can be a lot of things besides the conformer error that raises this ValueError. Just in case I think we should display the original ValueError message--that way if it is something else we can figure it out faster. Can you retain the message and pass it on to the first logging.error statement? Something like ```Quantum Mechanics calculation failed for species {0} with error {1}```?

Reply:
Should be fixed now